lib/plines/step.rb

```diff
@@ -65,6 +65,10 @@ def fan_out(&block)
       fan_out_blocks << block
     end
 
+    def run_if?(&block)
```

Comment:
Yeah. I completely agree. In fact, I thought about it while I was commuting -- that the name is not quite good. `skip_if` is a good suggestion.